test/test_nn.py

```diff
@@ -5303,11 +5305,11 @@ def update_shape(shape, dim, new_dim_size):
         # Correct activations
         for activation in activations:
             model = getattr(nn, model_name)(d_model, nhead, num_encoder_layers, num_decoder_layers,
-                                            dim_feedforward, dropout, activation)
+                                            dim_feedforward, batch_first, dropout, activation)
         # Incorrect activation
         with self.assertRaises(RuntimeError):
             model = getattr(nn, model_name)(d_model, nhead, num_encoder_layers, num_decoder_layers,
-                                            dim_feedforward, dropout, wrong_activation)
+                                            dim_feedforward, batch_first, dropout, wrong_activation)
```

Comment:
I think if you add batch_first to the end, you won't need to add it between other arguments.

Reply:
This is true, and I initially thought about doing that. However, `batch_first` has traditionally been an argument before `dropout` (see RNN docs), so that is my rational for putting it there.